style.css

```diff
@@ -1,3 +1,53 @@
 #root {
```

Comment:
I would suggest adding some formatting to the selectbox as when you use the search box it moves around as the size of the selectbox changes based on the available episodes.